static/js/init.js

```diff
@@ -3,6 +3,7 @@
 /* global require, window, document */
 
 var $ = require('jquery');
+var keyboard = require('keyboardjs');
```

Comment:
We don't need it, but copying and pasting key codes from the internet is extra work and error-prone. We're also using the library in the typeahead module, but not in depth--really, all it does is key mapping lookups and a more concise event listener. It's about 10k minified. This would be easy to revert if you want--I kind of like it, but no strong preferences.